src/@next/Switch/Switch.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ChangeEvent, useState } from 'react';
+import { InputStyle, LabelStyle, SwitchStyle } from './SwitchStyle';
+
+export const Switch = () => {
```

Comment:
It should be a controlled component and accept HTMLInput Props. This way, we can control the value (checked property) and add a custom function for the onChange event.

Reply:
Can refer to the `Checkbox` component implementation

Reply:
done